docs/modules/ROOT/pages/access_kubernetes_api.adoc

```diff
@@ -1,37 +0,0 @@
-== Access Kubernetes API
-
-=== K3S
-
-K3s’ installation create a `kubeconfig.yaml` file that contains the
-Kubernetes context that allows you to access the cluster.
-
-[source,shell]
-----
-export CLUSTER_NAME=master
-export KUBECONFIG=distributions/k3s/_/docker/terraform/terraform.tfstate.d/$CLUSTER_NAME/kubeconfig.yaml
-export BASE_DOMAIN=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' k3s-server-$CLUSTER_NAME|tr '.' '-'`.nip.io
-kubectl get nodes
-kubectl get namespaces
-kubectl get pods --all-namespaces
-----
-
-=== EKS
-
-. Make sure your AWS credentials file (`~/.aws/credentials` on Linux) contains the access key that has been used to create the cluster:
-+
-[source,toml]
-----
-[<ACCOUNT_NAME>]
-aws_access_key_id = <YOUR_ACCESS_KEY_ID>
-aws_secret_access_key = <YOUR_SECRET_ACCESS_KEY>
-region = <YOUR_REGION>
-----
-
-. Update your kubeconfig using the following command:
-+
-[source,shell]
-----
-aws --profile <ACCOUNT_NAME> eks --region <YOUR_REGION> update-kubeconfig --name <CLUSTER_NAME>
-----
-
-Then, you should be able to use Kubectl.
```

Comment:
This doc is important

Reply:
I find that this snippet would be better in a tutorial explaining what needs to be done to instanciate the stack on EKS.
Because, as-is, this page is unlinked to anything applies only to EKS and does little sense as a Howto since nowhere else there is a reference to EKS.

Reply:
Added a skeleton for this tutorial with this snippet in docs/modules/ROOT/pages/deploy_devops_stack_in_eks.adoc